src/pages/Home/components/ClubsTab.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { toast } from 'react-toastify';
+import { SearchIcon } from '@heroicons/react/solid';
+import OwnedTokenGrid from './OwnedTokenGrid';
+import { useTokenRegistry } from '../../../common/components/Solana/TokenRegistry';
+import GenerateTokenOwnedList from '../../../common/services/Solana/GetData/GenerateTokenOwnedList';
+import { userPublicKeyAtom } from '../../../recoil/userInfo';
+
+import { ItokenOwned } from '../../../common/types';
+
+export default function ClubsTab(): JSX.Element {
+  const userPublicKey = useRecoilValue(userPublicKeyAtom);
+  const tokenRegistry = useTokenRegistry();
+  const [tokenOwnedList, setTokenOwnedList] = useState<ItokenOwned[]>([]);
+
+  const [loading, setLoading] = useState(false);
+
+  async function generateList() {
+    setLoading(true);
+    if (userPublicKey !== undefined) {
+      try {
+        setTokenOwnedList(
+          await GenerateTokenOwnedList(tokenRegistry, userPublicKey)
+        );
+        /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+      } catch (error: any) {
+        toast.error(error?.message);
+      }
+    }
+    setLoading(false);
+  }
+  return (
+    <div>
+      {/* Search Bar  */}
+      <div className="flex-1 flex items-center justify-center mt-6">
+        <div className=" w-full ">
```

Comment:
remove whitespace around w-full.

Reply:
is this div and the div that says className="relative" necessary? I think maybe you can either remove both of them or combine them in a single div.

Reply:
okk